roxie/ccd/ccdstate.cpp

```diff
@@ -1727,11 +1753,11 @@ class CRoxiePackageSetWatcher : public CInterface
         reply.append("</PackageSets>\n");
     }
 
-    void getStats(StringBuffer &reply, const char *id, const char *action, const char *graphName, const IRoxieContextLogger &logctx) const
+    void getStats(StringBuffer &reply, const char *id, const char *graphName, bool x, const char *wuid, const IRoxieContextLogger &logctx) const
```

Comment:
bool x?

Reply:
Was just making sure that my change to the prototype would cause compile errors if I didn't find all callers... I can (and have) remove now.